arbitrum-docs/node-running/node-providers.mdx

```diff
@@ -26,6 +26,8 @@ This section provides an overview of the available public RPC endpoints for diff
 
 ⚠️ Unlike the RPC Urls, the Sequencer endpoints only support `eth_sendRawTransaction` and `eth_sendRawTransactionConditional` calls.
 
+⚠️ Important note: WebSocket calls are not supported when using the Arbitrum public endpoints.
```

Comment:
I'd say something like "Arbitrum public RPCs do not provide websocket support" to make it more clear.